x-pack/plugins/apm/public/utils/formatters.js

```diff
@@ -52,3 +52,14 @@ export function asInteger(value) {
 export function tpmUnit(type) {
   return type === 'request' ? 'rpm' : 'tpm';
 }
+
+export function getDurationPercent(
+  itemDuration: number,
+  totalDuration?: number,
+  fallbackResult?: string
+) {
+  if (totalDuration === undefined || totalDuration === 0) {
+    return fallbackResult || '';
```

Comment:
If we use a default prop for `fallbackResult` in the arguments list and default it to `''`, we won't have to worry about if the caller wants to fall back to `0` or `null` or something here.

Reply:
++